README.md

```diff
@@ -90,8 +111,16 @@ Define a function called `doSomething`. It should take one argument, called
 `thingToDo`. When called, `doSomething` should invoke the `thingToDo` function. Demonstrate calling `doSomething` while passing in an argument.
 
 Your Answer:
-```js
-// write code here
+```
+function doSomething(x, y){
+  if (x > 0) {
+    return y / x;
+  };
+  else {
+    return 'this is a negative number or undefined';
+  };
+};
+console.log(doSomething(5, 6))
```

Comment:
We expect `doSomething` to receive a reference to a function, which does something.  Please review our suggestion in the Solution branch.